pages/portfolio.html

```diff
@@ -1,12 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jennifer's Portfolio</title>
+    <link href="/styles/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <!-- nav menu -->
+    <header>
+      <h1 class="page-title"> Portfolio</h1>
+      <nav class="nav-menu">
+        <div>
+          <a href="/pages/index.html"> Home </a>
+        </div>
+        <div>
+          <a href="/pages/about.html"> About </a>
+        </div>
+      </nav>
+    </header>
+    
+    <!-- portfolio -->
+    <main class="portfolio">
+      <div id="project_1">
```

Comment:
Consider using a `section` for each of these, since they have a heading and additional content.